scoper.inc.php

```diff
@@ -120,6 +125,10 @@ function( $file_path, $prefix, $contents ) {
 				$contents = str_replace( "'Google_", "'" . $prefix . '\Google_', $contents );
 				$contents = str_replace( '"Google_', '"' . $prefix . '\Google_', $contents );
 			}
+			if ( false !== strpos( $file_path, 'apiclient-services-subscribewithgoogle' ) ) {
+				// Rewrite "Class_Name" to Class_Name::class to inherit namespace.
+				$contents = preg_replace( '/"(Google_[^"]+)"/', '\\1::class', $contents );
+			}
```

Comment:
This rewriting isn't needed for this package.

In the previous version, it was needed to handle these scenarios:

https://github.com/google/site-kit-wp/blob/92b79018b8e3a56a2dff2f3f45fa508bea7e993f/packages/apiclient-services-subscribewithgoogle/SubscribewithGoogle.php#L149

Whereas in the newly generated version, they are already using the `Class_Name::class` syntax:

https://github.com/google/site-kit-wp/blob/dfbf19413aebabae7b0852a6e8f3e2ebee34478f/packages/apiclient-services-subscribewithgoogle/SubscribewithGoogle/Resource/Publications.php#L54

```suggestion
```